pkgs/data/fonts/openmoji/default.nix

```
@@ -14,6 +17,34 @@ let
     [ "OpenMoji-Color.ttf" "OpenMoji-Black.ttf" ]
     variant;
 
+  # With newer fontforge the build hangs, see
+  # https://github.com/NixOS/nixpkgs/issues/167869
+  # Patches etc taken from
+  # https://github.com/NixOS/nixpkgs/commit/69da642a5a9bb433138ba1b13c8d56fb5bb6ec05
+  fontforge-20201107 = fontforge.overrideAttrs (old: rec {
```

```suggestion
  fontforge = fontforge.overrideAttrs (old: rec {
```

---

isn't that infinite recursion since in nix `let` is recursive?

---

🤔 could be. I am not sure right now.

Then normally something like the following is used

```suggestion
  fontforge' = fontforge.overrideAttrs (old: rec {
```

---

If the shadowing doesn't work I would rather stick with meaningful names rather than `'`.